zipper/zipper.cpp

```diff
@@ -287,9 +294,14 @@ namespace zipper {
                 delete m_impl;
 	}
 
+	bool Zipper::add(std::istream& source, const std::tm& timestamp, const std::string& nameInZip, zipFlags flags)
+	{
+		return m_impl->add(source, timestamp, nameInZip, m_password, flags);
+	}
+
 	bool Zipper::add(std::istream& source, const std::string& nameInZip, zipFlags flags)
 	{
-		return m_impl->add(source, nameInZip, m_password, flags);
+		return add(source, {}, nameInZip, flags);
```

Comment:
@Lecrapouille  {} is equivalent to std::tm tmp = {0}; so that it doesn't change existing behavior.